### DISCUSSION TITLE
My submission has stuck in the status of  evaluating for  over 10 hours

### DISCUSSION THREAD

**Discussion by pikaandhouse** [PaddlePaddle/PaddleSpatial#126] [BaiduKDDCup2022] [answered]
My submission has stuck in the status of  evaluating for 
over 10 hours .My team name is  asdzwl and my submit time is  2022-05-31 10:50.

**Comment by BruceQD** [comment]
I encountered the same problem. My submission has been submitted for over 4 hours, while it just ran about half an hour with my laptop. And my previous submission on May 25th just ran for about 20 minutes online.
I guess the online evaluation system is working abnormally.

**Reply by AlvinAi96** [reply]
Same as you. Upvote the question. Waiting for the reply from the staff.

**Comment by ramber1836** [comment]
The hard workload of the online test environment might be the reason.

**Reply by ChaoqunSu** [reply]
> The hard workload of the online test environment might be the reason.

Can you solve this problem quickly?  My submission has stuck in the status of evaluating for over 5 hours

**Comment by Hiutin** [comment]
My submissions also have the same problem of waiting for over 10 hours. Would the organizers quickly fix it?

**Comment by fersebas** [comment]
My submission is also stuck :S. Team datateam-UCM.

**Comment by ChaoqunSu** [comment]
I feel Baidu is not doing well in this competition  as arganizers，hope the staff can fix the problem as soon as possible

**Comment by ramber1836** [comment, marked as answer]
Apologies for the inconvenience. 
Many competitions have been launched recently on the AIStudio which brings high overload for the backend system. KDDCup is one of those competitions. 
Extra resources have been allocated, the 'keep-evaluating' issues will be alleviated.

**Reply by wang975679801** [reply]
Will the computation time cost be the same regardless of when the submission is made? Otherwise, it maybe unfair to competitors in different time zones. (Such as the AIStudio is busy during the day time but free duiring the night.)

**Reply by ramber1836** [reply]
This is a common issue in different competitions. 
A reasonable solution to tackle such an issue is to allocate more resources. That's what we trying to do.
